.github/workflows/release.yml

```diff
@@ -7,17 +7,31 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+  release:
     runs-on: ubuntu-latest
```

Comment:
Do we need such a "heavy" image?

Reply:
This is the recommended image according to https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven

Reply:
Oke